### ISSUE TITLE
The README for youtube-dl mandates the incorrect line ending convention for macOS

### ISSUE THREAD

**Issue by Saklad5** [rg3/youtube-dl#11709]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
- Use *Preview* tab to see how your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2017.01.14*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I’ve **verified** and **I assure** that I'm running youtube-dl **2017.01.14**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [x] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### Description of your *issue*, suggested solution and other information

The README page for youtube-dl states 

> Make sure you have correct newline format in the cookies file and convert newlines if necessary to correspond with your OS, namely CRLF (\r\n) for Windows, LF (\n) for Linux and CR (\r) for Mac OS.

However, this has not been true for over fifteen years, far before this project, YouTube, GitHub, or even Git existed. While Mac OS 9 did use \r as a newline character, macOS has always followed the UNIX convention of \n since its 2001 release. I’m honestly baffled as to how this mistake occurred.

To summarize: Windows uses \r\n, and **_everything else_** uses \n.
